lib/gen-core.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs')
```

Comment:
Can we have a separate PR for this change @sumanthyedoti ?

Reply:
Yeah, I remember Santosh saying that minification is not required (mostly because it makes the generated code hard to read, and also minification can be handled by some other tool). 

Maybe we can have a CLI flag to minify this, but we should make it a separate PR. Let's also check with Santosh about this.
For now this can be removed.